docs/shapes/Shape_Constructors.rst

```diff
@@ -110,6 +110,10 @@ Shape Constructors
     Right hand palm on p1, fingertips on p2, thumb is the normal vector
     (points away from the half-plane).
 
+``curve.quadratic p0 p1 p2``
```

Comment:
This doesn't belong here, since `curve.quadratic` doesn't work unless you have included `lib.curves`, which isn't mentioned. Since this is actually documentation for `lib.curves`, it belongs in `docs/lib/Curves.rst`.